prow/jobs/busola/busola-web/periodic-performance-tests-web.yaml

```diff
@@ -0,0 +1,15 @@
+# Periodic jobs
+
+# Description- This prow job is for running performance Tests on kyma web application. 
+periodics:
+- interval: 24h
+  agent: kubernetes
```

Comment:
I don't think you need to define ProwJob agent here. We use Kubernetes agent by default.

Reply:
Added- `decorate: true` and removed `agent: kubernetes`